packages/shared/lib/services/file/remote.service.ts

```diff
@@ -12,7 +12,7 @@ import localFileService from './local.service.js';
 
 let client: S3Client | null = null;
 
-if (isCloud()) {
+if (isCloud() || isLocal()) {
```

Comment:
isn't this equivalent to always true since isCloud = isProd + isStaging

Reply:
enterprise

Reply:
what about making it clearer with a condition `if(!isEnterprise)`